lib/game.rb

```diff
@@ -3,23 +3,40 @@ class Game
 
   def initialize
     @board = Board.new
-    @players = []
+    @players = create_players
+    @message_bot = Message.new
```

Comment:
I like the changes to have game instantiate all the other class objects we'll need when it initializes. This lets it have easy access to their methods as well.